firebase-installations-interop/firebase-installations-interop.gradle

```diff
@@ -40,7 +40,7 @@ android {
 
 dependencies {
 	implementation 'com.google.android.gms:play-services-tasks:18.0.1'
```

Comment:
quick nit, if you could fix the indentation, :-)

Reply:
Done